ray-operator/controllers/ray/rayjob_controller.go

```diff
@@ -333,9 +335,16 @@ func (r *RayJobReconciler) Reconcile(ctx context.Context, request ctrl.Request)
 				logger.Info(fmt.Sprintf("shutdownTime not reached, requeue this RayJob for %d seconds", delta))
 				return ctrl.Result{RequeueAfter: time.Duration(delta) * time.Second}, nil
 			}
-			// We only need to delete the RayCluster. We don't need to delete the submitter Kubernetes Job so that users can still access
-			// the driver logs. In addition, a completed Kubernetes Job does not actually use any compute resources.
-			if _, err = r.deleteClusterResources(ctx, rayJobInstance); err != nil {
+			if os.Getenv(DELETE_RAYJOB_CR_AFTER_JOB_FINISHES) == "true" {
+				err = r.Client.Delete(ctx, rayJobInstance)
+				logger.Info("RayJob CR is deleted", "RayJob", rayJobInstance.Name)
```

Comment:
I think every log message should already include the RayJob CR's name. Would you mind checking it? We can remove `"RayJob", rayJobInstance.Name` if it is redundant.

Reply:
Done

Reply:
Finally I removed this logging because I found there is already another place to log the RayJob deletion event.
https://github.com/ray-project/kuberay/blob/b1f721a30de8166f0c0505569829fbf88fdcc8e5/ray-operator/controllers/ray/rayjob_controller.go#L95
![image](https://github.com/ray-project/kuberay/assets/47914085/069160f2-5d75-42d8-83d3-0bac94d889e7)

Reply:
I might also prefer to add a log message here because we can't ensure that:

* The RayJob will always have a finalizer added in the future. I seldom hear about use cases for the finalizer, so I might make it optional in the future.
* The RayJob won't be requeued again. We can't promise that it will always have another reconciliation.

Reply:
Got it. I've added back the logging.